pipelines/open-edx/forum_docker_build_pipeline.yaml

```diff
@@ -0,0 +1,61 @@
+---
+resource_types:
+- name: github-release
+  type: registry-image
+  source:
+    repository: concourse/github-release-resource
+
+resources:
+- check_every: 24h
+  name: forum-release
+  type: github-release
```

Comment:
This won't be a github-release resource type because that repo doesn't generally use the releases, instead we'll need to use the `git` resource type and trigger when there is a new commit on a given branch.

Reply:
actually i tested this with https://github.com/openedx/cs_comments_service/releases